### PR TITLE
Fix broken links - Part 10

### PR DESCRIPTION
**Pull request by btat**
This part of a series of PRs to fixe the broken links discovered after adding a link checker in #208.

Note that a small fraction (~15) of the 180 warnings still remain:
- Alternative/new links could not be found for some external links. E.g. https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers
- Docusaurus supports setting [custom heading IDs](https://docusaurus.io/docs/markdown-features/toc#heading-ids), which get flagged as broken by the checker.
- We've manually added anchors directly with HTML to some areas that normally don't have them. These get flagged as broken by the checker.

### CODE REVIEW

File: docs/pages-for-subheaders/manage-clusters.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ After you provision a cluster in Rancher, you can begin using powerful Kubernete
 
 This page covers the following topics:
 
-- [Switching between clusters](#switching-between-clusters)
 - [Managing clusters in Rancher](#managing-clusters-in-rancher)
 - [Configuring tools](#configuring-tools)
```
Comment (martyav) [inline docs/pages-for-subheaders/manage-clusters.md:10]: So we still need the table of contents?

File: versioned_docs/version-2.6/pages-for-subheaders/manage-clusters.md [hunk 2]
```diff
@@ -6,7 +6,6 @@ After you provision a cluster in Rancher, you can begin using powerful Kubernete
 
 This page covers the following topics:
 
-- [Switching between clusters](#switching-between-clusters)
 - [Managing clusters in Rancher](#managing-clusters-in-rancher)
 - [Configuring tools](#configuring-tools)
```
Comment (martyav) [inline versioned_docs/version-2.6/pages-for-subheaders/manage-clusters.md:10]: Similar to comment on other version of page

**Review by martyav — CHANGES_REQUESTED** [review]
Left some comments about a ToC on two pages, and restructured one section so the link could appear within a sentence describing the feature, instead of having a "See x for more details)

File: docs/pages-for-subheaders/manage-clusters.md (hunk 1 above)
Comment (btat) [reply docs/pages-for-subheaders/manage-clusters.md]: We can forego them. Removed in 1c0ef1d5c16da08fd39bc39fee15983620b4d475.

File: versioned_docs/version-2.6/pages-for-subheaders/manage-clusters.md (hunk 2 above)
Comment (btat) [reply versioned_docs/version-2.6/pages-for-subheaders/manage-clusters.md]: Removed in 1c0ef1d5c16da08fd39bc39fee15983620b4d475.

**Review by martyav — APPROVED** [review]

**Pull request merged**